cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -794,6 +794,27 @@ const char* gdv_fn_initcap_utf8(int64_t context, const char* data, int32_t data_
   *out_len = out_idx;
   return out;
 }
+
+GANDIVA_EXPORT
+int32_t gd_fn_instr_utf8(int64_t context, const char* string, int32_t string_len,
+                         const char* substring, int32_t substring_len) {
```

Comment:
If the substring is a literal, you should be able to pre-process the substring to implement this faster. Can you open a separate ticket to handle a literal substring?

Reply:
Yes, no problem. Should I open it before or after this one is merged?

Reply:
After this one is merged is fine